hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/terasort/PravegaGlobalOrderOutputFormat.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.pravega.example.hadoop.terasort;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.pravega.client.ClientFactory;
+import io.pravega.client.admin.StreamManager;
+import io.pravega.client.stream.*;
+import io.pravega.example.hadoop.wordcount.PravegaOutputFormat;
+import io.pravega.example.hadoop.wordcount.PravegaOutputRecordWriter;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.mapreduce.*;
```

Comment:
Can you please import specific classes instead of *?

Reply:
fixed